vulnerabilities/tests/test_data_source.py

```diff
@@ -41,249 +36,6 @@ def load_oval_data():
     return etrees_of_oval
 
 
-@pytest.fixture
```

Comment:
Thanks for removing old tests, IMO we should have some new tests to check the new GitImporter

Reply:
@ziadhany gentle ping!

Reply:
Sorry I was sick 🤧, I'm working on it